integration-tests/esm.spec.js

```diff
@@ -9,8 +9,15 @@ const {
 const path = require('path')
 const { assert } = require('chai')
 
+const NODE_MAJOR = parseInt(process.versions.node.split('.')[0])
+
 const hookFile = 'dd-trace/loader-hook.mjs'
 
+// TODO: add ESM support for Node 20 in import-in-the-middle
+const describe = NODE_MAJOR >= 20
```

Comment:
Why not use `semver.satisfies(...)` here?

Reply:
It didn't seem necessary for something that easy.